src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -113,6 +113,11 @@ public function action_handle_verify_webhooks() : void {
 	 */
 	public function handle_stripe_errors() {
 
+		// Bail if this is an ajax call.
```

Comment:
Should we update the `@since` in the docblock?

Reply:
Good catch. I'll add one.